lib/tools/adb-emu-commands.js

```diff
@@ -65,23 +65,30 @@ emuMethods.SENSORS = {
  * @returns {Array<EmuInfo>}
  */
 async function listEmulators () {
-  // https://android.googlesource.com/platform/sdk/+/33f330d963c1bf24667eddb891531abcce9fb4de/sdkmanager/libs/sdklib/src/com/android/sdklib/internal/avd/AvdManager.java
+  let avdsRoot = process.env.ANDROID_AVD_HOME;
+  if (await dirExists(avdsRoot)) {
+    return mapAvdDir(avdsRoot);
+  }
+
+  if (avdsRoot) {
+    log.debug(`Value of $ANDROID_AVD_HOME '${location}' does not exist`);
+  }
+
   const prefsRoot = await getAndroidPrefsRoot();
   if (!prefsRoot) {
     return [];
   }
 
-  const avdsRoot = path.resolve(prefsRoot, 'avd');
-  if (!await fs.exists(avdsRoot)) {
+  avdsRoot = path.resolve(prefsRoot, 'avd');
+  if (!await dirExists(avdsRoot)) {
+    log.debug(`Virtual devices config root '${avdsRoot}' is not an existing directory`);
     return [];
   }
 
-  const stats = await fs.stat(avdsRoot);
-  if (!stats.isDirectory()) {
-    log.debug(`Virtual devices config root '${avdsRoot}' is not a directory`);
-    return [];
-  }
+  return mapAvdDir(avdsRoot);
+}
 
+function mapAvdDir (avdsRoot) {
```

Comment:
async function

Reply:
how about renaming it to `getAvdConfigPaths`?

Reply:
Red CI was here

```
Message:
    /Users/runner/work/1/s/lib/tools/adb-emu-commands.js: 'await' is only allowed within async functions (92:18)

  90 | 
  91 | function mapAvdDir (avdsRoot) {
> 92 |   const configs = await fs.glob('*.ini', {
     |                   ^
  93 |     cwd: avdsRoot,
  94 |     absolute: true,
  95 |   });
```